awsutil/awsutil.go

```
@@ -56,6 +56,6 @@ func TimeDifference(server string) (time.Duration, error) {
 		return time.Duration(0), errors.New("invalid ntpdate output")
 	}
 
-	f, err := strconv.ParseFloat(string(submatched[1]), 64)
+	f, _ := strconv.ParseFloat(string(submatched[1]), 64)
```

@shurcooL Original code also ignore error.
So, I use a "_" variable.